charts/mattermost-operator/templates/mysql-operator/statefulset.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Values.mysqlOperator.appName }}-operator
+  namespace: {{ .Values.mysqlOperator.appName }}-operator
+  labels:
+    app: {{ .Values.mysqlOperator.appName }}-operator
+    release: {{ .Values.mysqlOperator.appName }}-operator
+spec:
+  replicas: {{ .Values.mysqlOperator.replicas }}
+  serviceName: {{ .Values.mysqlOperator.appName }}-operator-orc
+  podManagementPolicy: Parallel
+  selector:
+    matchLabels:
+      app: {{ .Values.mysqlOperator.appName }}-operator
+      release: {{ .Values.mysqlOperator.appName }}-operator
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.mysqlOperator.appName }}-operator
+        release: {{ .Values.mysqlOperator.appName }}-operator
+{{- if .Values.mysqlOperator.statefulSet.annotations }}
+      annotations:
+        {{- toYaml .Values.mysqlOperator.statefulSet.annotations | nindent 8 }}
+{{- end }}
+    spec:
+      serviceAccountName: {{ .Values.mysqlOperator.appName }}-operator
```

Comment:
it would be a good idea to have this ready in `helper.tpl` and just use the variable, so there is one place where this suffix is appended.

Reply:
Did a change of both name and namespace everywhere. They are now templated in tpl ;)